src/apps/external_executor/external_executor.cpp

```diff
@@ -47,12 +48,99 @@ namespace externalexecutor
         .install();
     }
 
+    // Note: As a temporary solution for testing, this app stores a single Tx,
+    // stolen from a StartTx RPC rather than a real client request
```

Comment:
It looks like the endpoint actually steals the client request, from `ctx`?

Reply:
Terminology is fuzzy here, but `StartTx` is (eventually) from an executor, not a 'real client'. Currently, as a PoC, we're storing a single `Tx`, and stealing that from `StartTx`. Eventually, we'll be storing many `Tx`s, each stolen from a user request to any other endpoint (eg - a call to `POST /students/{id}`)